flytekit/types/directory/__init__.py

```diff
@@ -10,13 +10,14 @@
    :template: file_types.rst
 
    FlyteDirectory
+   FlyteDirToMultipartBlobTransformer
    TensorboardLogs
    TFRecordsDirectory
 """
 
 import typing
 
-from .types import FlyteDirectory
+from .types import FlyteDirectory, FlyteDirToMultipartBlobTransformer
```

Comment:
to import `FlyteDirToMultipartBlobTransformer` in the pydantic plugin, we have to import here.